examples/autoscaler-webhook/Makefile

```diff
@@ -27,7 +27,7 @@ REPOSITORY = gcr.io/agones-images
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 project_path := $(dir $(mkfile_path))
-autoscaler_webhook_tag = $(REPOSITORY)/autoscaler-webhook:0.1
+autoscaler_webhook_tag = $(REPOSITORY)/autoscaler-webhook:0.1.1
```

Comment:
I assume you need me to push this up?

Happy for it to be 0.2 - that's what we've done for our other exampes.

Reply:
Yes, I would be happy if you can push 0.2 new version, I will update it here and in E2E test.